nifi-registry-core/nifi-registry-data-model/src/main/java/org/apache/nifi/registry/bucket/Bucket.java

```diff
@@ -41,6 +41,8 @@
 
     private String description;
 
+    private Boolean allowExtensionBundleRedeploy;
```

Comment:
Instead of a boolean flag here. I'm wondering if this would be better as a Enum for something like "redeploy policy" with values such as "NEVER, SNAPSHOTS, ALWAYS" or something like that?

Reply:
Ok, I’m not familiar with Nexus’s configuration options, but I’m in favor of keeping consistencies/parallels with that. I’m good with leaving this as-is.